jobs/dynatrace-oneagent/templates/pre-start.erb

```diff
@@ -110,12 +110,12 @@ setHostProps() {
 downloadAgent() {
     local downloadUrl=$1
     local installerFile=$2
-    local iOriginHeader='x-dynatrace-iorigin:bosh'
+    local userAgent="bosh/<%= spec.release.version %>"
 
     if curl -h &> /dev/null ; then
-        local downloadCommand="curl -s -S -f --connect-timeout 10 $SSL_INSECURE_CURL ${downloadUrl} -o ${installerFile} --header ${iOriginHeader}"
+        local downloadCommand="curl -s -S -f --connect-timeout 10 $SSL_INSECURE_CURL ${downloadUrl} -o ${installerFile} -A ${userAgent}"
```

Comment:
Please always use the long version of a command line flag/argument in scripts :-)
In this case `--user-agent` instead of `-A`